### ISSUE TITLE
HUD not showing up [BUG]

### ISSUE THREAD

**Issue by dharkus5000** [Larkinabout/fvtt-token-action-hud-core#165]
**Describe the bug**
HUD doesn't show up

**Steps to reproduce**
possibly? have both classic & core installed at once, then even if you disable classic it doesn't work? not sure if that's related?

**Versions:**
- Token Action HUD Core: latest and possibly last version too?
- Token Action HUD System: 2e latest
- System: pf2e latest
- Foundry VTT: latest
- Browser: various

**Additional context**
got this error
`Error: An error occurred while rendering TokenActionHud 157. ENOENT: no such file or directory, open '/forge-vtt/data/Data/modules/token-action-hud-core/templates/template.hbs'
[No packages detected]
    getTemplate https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/foundry.js:7647
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    emit https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
foundry.js:753:30`
& this one
`Error: An error occurred while rendering TokenActionHud 157. ENOENT: no such file or directory, open '/forge-vtt/data/Data/modules/token-action-hud-core/templates/template.hbs'
[No packages detected]
    getTemplate https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/foundry.js:7647
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    emit https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
    value https://voqu-blood-lords-friday.eu.forge-vtt.com/scripts/socket.io.min.js:6
foundry.js:753:30
`
apparently same one twice?

**Comment by Larkinabout** [comment]
Try clearing your browser cache.

**Comment by dharkus5000** [comment]
i did ctrl+F5 reload, no change

**Comment by dharkus5000** [comment]
no idea if this is related, but on a different world to the other one that had the same circumstances, i see the HUD, but get this error every time i select any token
```
Error: An error occurred while rendering TokenActionHud 154. One of original or other are not Objects!
[Detected 1 package: token-action-hud-core]
    mergeObject https://voqu-stolen-fate.eu.forge-vtt.com/scripts/commons.js:2673
    _render https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:5819
    render https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:5742
    #at https://assets.forge-vtt.com/bazaar/modules/token-action-hud-core/1.4.13/scripts/token-action-hud-core.min.js:1
    update https://assets.forge-vtt.com/bazaar/modules/token-action-hud-core/1.4.13/scripts/token-action-hud-core.min.js:1
    <anonymous> https://assets.forge-vtt.com/bazaar/modules/token-action-hud-core/1.4.13/scripts/token-action-hud-core.min.js:1
    #call https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:730
    callAll https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:687
    control https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:28434
    _onClickLeft https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:28823
    _onClickLeft https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:56015
    callback https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:31547
    #handleClickLeft https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:31734
    #handleMouseDown https://voqu-stolen-fate.eu.forge-vtt.com/scripts/foundry.js:31719
    notifyListeners https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    notifyTarget https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    propagate https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    dispatchEvent https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    mapPointerDown https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    mapEvent https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
    onPointerDown https://voqu-stolen-fate.eu.forge-vtt.com/scripts/pixi.min.js:869
foundry.js:753:30
```

**Comment by Larkinabout** [comment]
Sorry, I should have said update to 1.4.14 first.

**Comment by brandonhinds** [comment]
My DM had these exact same symptoms and clearing the cache worked. Thanks @Larkinabout .

**Comment by unwashedotaku** [comment]
Hi, I'm a DM and I am also having these issues and I cleared my cache and this did not fix the issue.

**Comment by unwashedotaku** [comment]
> Hi, I'm a DM and I am also having these issues and I cleared my cache and this did not fix the issue.

![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/140227987/487fee84-2940-416d-bc2c-a3b30821854e)
Here's a copy of the Error out of chrome's Dev tools.

**Comment by lammonaaf** [comment]
I've also started receiving these errors (hud does not show up too). It started today, when right after I've upgraded pf2e system to 5.2. Also, we've experienced "actions" and "spells" sections disappearing even before the update.

**Comment by brandonhinds** [comment]
I should mention I'm playing in a 5e game (where clearing the cache worked), and I just realised this ticket is for a Pathfinder 2e game.

**Comment by failwiz** [comment]
Had the same problem, and after updating pf2e system to 5.2.1 it works again. Had the same error message as @unwashedotaku, but now it gives this:

```
Error: CONFIG.PF2E.attributes.perception is deprecated. Use localization path PF2E.PerceptionLabel directly instead.
Deprecated since Version 5.2.0
Backwards-compatible support will be removed in Version 6.0.0
    at Object.logCompatibilityWarning (commons.js:1993:19)
    at notifyDeprecatedPath (pf2e.mjs:138:17291)
    at get perception [as perception] (pf2e.mjs:138:18140)
    at #u (token-action-hud-pf2e.min.js:1:25514)
    at #t (token-action-hud-pf2e.min.js:1:16337)
    at ActionHandler.buildSystemActions (token-action-hud-pf2e.min.js:1:16080)
    at #c (token-action-hud-core.min.js:1:27068)
    at ActionHandler.buildHud (token-action-hud-core.min.js:1:25318)
    at async #at (token-action-hud-core.min.js:1:84634)
```

**Comment by unwashedotaku** [comment]
> Had the same problem, and after updating pf2e system to 5.2.1 it works again. Had the same error message as @unwashedotaku, but now it gives this:
> 
> ```
> Error: CONFIG.PF2E.attributes.perception is deprecated. Use localization path PF2E.PerceptionLabel directly instead.
> Deprecated since Version 5.2.0
> Backwards-compatible support will be removed in Version 6.0.0
>     at Object.logCompatibilityWarning (commons.js:1993:19)
>     at notifyDeprecatedPath (pf2e.mjs:138:17291)
>     at get perception [as perception] (pf2e.mjs:138:18140)
>     at #u (token-action-hud-pf2e.min.js:1:25514)
>     at #t (token-action-hud-pf2e.min.js:1:16337)
>     at ActionHandler.buildSystemActions (token-action-hud-pf2e.min.js:1:16080)
>     at #c (token-action-hud-core.min.js:1:27068)
>     at ActionHandler.buildHud (token-action-hud-core.min.js:1:25318)
>     at async #at (token-action-hud-core.min.js:1:84634)
> ```

Can confirm the same.  It's back, the warnings are pertaining to what appears to be using the perception attribute or locating the pathway to it.  It's just a warning that it's been depreciated and won't be working eventually whenever the Core PF2e team removes that pathway.

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Pathfinder 2 1.4.12.
